src/database.ts

```diff
@@ -0,0 +1,138 @@
+import firebase from "firebase/app";
+import "firebase/database";
+const crypto = require('crypto')
+
+type blankPassObject =  {
+    ConnectToHubspot: boolean,
+    CreateParticipant: boolean,
+    Field: boolean,
+    First: boolean,
+    Last: boolean,
+    SEmail: boolean,
+    Refferable: boolean,
+    DeleteWhenDeleted: boolean,
+    ConnectToSaasquach: boolean,
+    CreateInHubspot: boolean,
+    ContactField: boolean,
+    Name: boolean,
+    HEmail: boolean,
+    ContactOwner: boolean,
+    AssosiatedCompany: boolean,
+    LastActivityDate: boolean,
+    CreateDate: boolean,
+    DeleteConnected: boolean,
+    ConnectShareLinks: boolean,
+    AddShareLinks: boolean,
+};
+
+//import { AddToDatabase } from "../database"//How to use
+//abc();
+function hashValue(stringValue: string){
+    stringValue.toLowerCase()
+    return crypto.createHash('sha1').update(stringValue).digest('hex');
+}
+
+/**
+ * Adds Values to Database
+ */
+export function AddToDatabase(email: string, data: blankPassObject) {
```

Comment:
We could probably use the Configuration type here (in types.ts) and get rid of the blankPassObject type altogether